Makefile

```diff
@@ -295,4 +295,4 @@ release-manifest:
 promote-staging-manifest: #promote staging manifests to release dir
 	$(MAKE) release-manifest
 	@rm -rf charts/secrets-store-sync-controller/
-	@cp -r manifest_staging/charts ./charts
+	@cp -r manifest_staging/charts/ ./charts
```

Comment:
This will publish charts in `charts/secrets-store-sync-controller`. Earlier chart release job was failing since it was being publish at `charts/charts/secrets-store-sync-controller`.